updated commit with 3 tests

TregubovYury opened the pull request:
Author: Tregubov Yury

Made new commit with code of QA-450 QA-456 and QA-458
Including all apllied comments from
https://github.com/Mirantis/mos-integration-tests/pull/52

Commit:
updated commit with 3 tests

Author: Tregubov Yury

Made new commit with code of QA-450 QA-456 and QA-458
Including all apllied comments from
https://github.com/Mirantis/mos-integration-tests/pull/52

Commit:
updated commit with 3 tests

Author: Tregubov Yury

Made new commit with code of QA-450 QA-456 and QA-458
Including all apllied comments from
https://github.com/Mirantis/mos-integration-tests/pull/52

rpromyshlennikov commented on mos_tests/neutron/python_tests/test_dhcp_agent.py:
```diff
@@ -55,15 +49,40 @@ def prepare_openstack(self, setup):
-                                       binary='neutron-dhcp-agent')['agents']]
-        return dhcp_agts
+        self.dhcp_agent_ids = [agt['id'] for agt in
+                                self.os_conn.neutron.list_agents(
```
It seems like one space is unnecessary, pep-8: visual identation

TregubovYury replied:
All checks are passed so far with that space. Did not find any problems with manual flake8 execution as well.

TimurNurlygayanov replied:
let's fix this space, line 53.

gdyuldin committed:
Add testcase 'Broadcast traffic between instances'

Testcase checks broadcast traffic between instances placed in a single
private network and hosted on different nodes

Scenario:
1. Create private network net01, subnet 10.1.1.0/24
2. Create private network net02, subnet 10.1.2.0/24
3. Create router01_02 and connect net01 and net02 with it
4. Boot instances vm1 in net01 and compute1
5. Boot instances vm2 in net01 and compute2
6. Boot instances vm3 in net02 and compute2
7. On the compute2 start listen traffic from vm1 fixed ip on
    tap interface of vm3
8. Go to the vm1's console and initiate broadcast traffic to vm3
9. Check that ARP traffic appears on listened interface
10. On the compute2 start listen traffic from vm1 fixed ip om
    tap interface of vm2
11. Go to the vm1's console and initiate broadcast traffic to vm3
12. Check that ARP traffic is absent on listened interface

Commit:
Added test test_ovs_restart_pcs_disable_enable.

Restarting ovs-agents with pcs disable/enable on controllers.

Steps:
1. Update default security group
2. Create router01, create networks net01: net01__subnet,
    192.168.1.0/24, net02: net02__subnet, 192.168.2.0/24 and
    attach them to router01.
3. Launch vm1 in net01 network and vm2 in net02 network
    on different computes
4. Go to vm1 console and send pings to vm2
5. Disable ovs-agents on all controllers, restart service
    neutron-plugin-openvswitch-agent on all computes, and enable
    them back. To do this, launch the script against master node
6. Check that pings between vm1 and vm2 are not interrupted or
    not more than 2 packets are lost

Commit:
fixed comment for the #65

Author: Tregubov Yury

Commit:
fixed comment for the #65

Author: Tregubov Yury

Commit:
removed some more spaces

Author: TregubovYury

Commit:
removed some more spaces

Author: TregubovYury

rpromyshlennikov committed:
Typo fixed.
Update test ban dhcp multiple times duration.

Commit:
Fixes for test_ovs_restart_pcs_disable_enable.

Commit:
Modified scenario for test.

In test_ovs_restart_pcs_disable_enable ovs_agents should be disabled only
on one controller.

Commit:
Changed test scenatio

Restart openvswitch-agents with pcs disable/enable on controllers

Steps:
1. Update default security group
2. Create router01, create networks net01: net01__subnet,
192.168.1.0/24, net02: net02__subnet, 192.168.2.0/24 and
attach them to router01.
3. Launch vm1 in net01 network and vm2 in net02 network
on different computes
4. Go to vm1 console and send pings to vm2
5. Disable ovs-agents on all controllers, restart service
neutron-plugin-openvswitch-agent on all computes, and enable
them back. To do this, launch the script against master node.
6. Wait 30 seconds, send pings from vm1 to vm2 and check that
it is successful.

gdyuldin committed:
Add posibility to execute command on instance

New method allow to get direct SSHClient to instance
Benefits:
1. Check command return code
2. check stdout and stderr directly
3. Doesn't require floating ip requirements

Commit:
Applying comments for test.

Commit:
"Applying comments. Changed env checks."

For OvsBase is checked that one ore more computes exist in the env as it
can be required to deploy env with 1 compute for similar tests.
For TestOVSRestartTwoVms is checked that two or more computes exist in the env
as we should create 2 vms on 2 computes.

rpromyshlennikov committed:
Move to direct ssh connection for vms.

rpromyshlennikov committed:
Style fixes in os_actions.

rpromyshlennikov committed:
Add test manual reschedule dhcp agent.
Add function to get ports, add and remove network from dhcp agent in os action.

Commit:
Merge pull request #67 from MikeLaptev/windows_tests

Windows tests

Commit:
Merge pull request #72 from rpromyshlennikov/feature/QA-426-Reschedule-dhcp-agent

Feature/qa 426 reschedule dhcp agent

Commit:
Merge pull request #71 from gdyuldin/remote

Add posibility to execute command on instance

Commit:
Merge pull request #69 from gdyuldin/QA-364

Add testcase 'Broadcast traffic between instances'

Commit:
Merge pull request #68 from AlexGromov/neutron

Added test test_ovs_restart_pcs_disable_enable.

Commit:
Merge branch 'master' of https://github.com/TregubovYury/mos-integration-tests

Conflicts:
	mos_tests/environment/os_actions.py
